pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -90,7 +90,7 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 				request := &elb.AttachLoadBalancerToSubnetsInput{}
 				request.LoadBalancerName = aws.String(loadBalancerName)
 				request.Subnets = stringSetToPointers(additions)
-				glog.V(2).Info("Attaching load balancer to added subnets")
+				glog.V(2).Infof("Attaching load balancer for %v to added subnets %v", request.LoadBalancerName, request.Subnets)
```

Comment:
Well, in my case I could see that there were six pointers instead of three and that the first three were null. But yeah, backporting is probably a bigger deal than more verbose logging in an obscure case.